src/Pages/Platforms/platformInfo.tsx

```diff
@@ -2,29 +2,26 @@ import * as React from "react"
 import { RouteComponentProps } from "react-router-dom"
 
 import { ErddapObservationTable, ErddapPlatformGetter, ErddapPlatformInfoPanel, PlatformAlerts } from "Features/ERDDAP"
-import { UnitSelector } from "Features/Units"
+import { UnitSelector, useUnitSystem } from "Features/Units"
 
 import { PlatformMatchParams } from "./types"
 
 /**
  * Display our platform info panel for the select platform.
  */
-export class PlatformInfo extends React.Component<RouteComponentProps, object> {
-  public render() {
-    const { id } = this.props.match.params as PlatformMatchParams
+export const PlatformInfo: React.FunctionComponent<RouteComponentProps> = ({ match }) => {
+  const { id } = match.params as PlatformMatchParams
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)